src/compat/tracker.jl

```diff
@@ -460,3 +469,25 @@ end
 
     return α, find_alpha_pullback
 end
+
+# `OrderedBijector`
+Bijectors._transform_ordered(y::Union{TrackedVector,TrackedMatrix}) = track(Bijectors._transform_ordered, y)
+@grad function Bijectors._transform_ordered(y::AbstractVecOrMat)
+    x, dx = ChainRulesCore.rrule(Bijectors._transform_ordered, data(y))
+    return x, (wrap_chainrules_output ∘ Base.tail ∘ dx)
+end
```

Comment:
Do you know if this is "dangerous" in any way @devmotion ? Just seems a bit "too easy".

Reply:
Not sure, seems similar to what Zygote does. But the conversion of ChainRules types there is a bit tricky, so I assume also here it could be tricky to implement `wrap_chainrules_output` correctly (e.g. one should not drop NotImplemented and unthunk stuff). But as long as you only apply it to rules that you've defined it is fine I guess. For simple operations the possible overhead might be noticeable but I guess it doesn't matter for expensive operations.

(BTW we already use `rrule` in definitions for other ADs in DistributionsAD: https://github.com/TuringLang/DistributionsAD.jl/blob/fe2070012167b78c84a49733a0a64997e9533812/src/common.jl#L8-L13)

Reply:
> Not sure, seems similar to what Zygote does. But the conversion of ChainRules types there is a bit tricky, so I assume also here it could be tricky to implement wrap_chainrules_output correctly (e.g. one should not drop NotImplemented and unthunk stuff). But as long as you only apply it to rules that you've defined it is fine I guess. For simple operations the possible overhead might be noticeable but I guess it doesn't matter for expensive operations.

I was looking at what Zygote was doing, and it doesn't seem like it would be incredibly difficult to make something like `@defer_chainrules` or something for Tracker.jl and ReverseDiff.jl, but for the reasons you mentioned I decided to not be overly ambitious right now.

And I was thinking the same: should be fine since in these cases.

> BTW we already use rrule in definitions for other ADs in DistributionsAD

Oh nice! Wasn't aware :+1: